test/checks/forms/submit-button.js

```diff
@@ -0,0 +1,117 @@
+describe('submit-button', function() {
```

Comment:
These tests don't properly describe what's going on in the check. There are only a few things happening in this check that need to be tested:

- Form has a submit button that isn't disabled
- Form has no submit button, and no fields
- Form has no submit button, and a single input element
- Form has no submit button, all other scenarios

If you want to test the individual selectors, you can put them in an array and loop each of them into a describe group. Something like this:

```js
['<input type="submit" />', '<button></button>', ...].forEach(function (buttonText) {
	describe(buttonText, function () {
		...
	})
});
```